Cargo.toml

```diff
@@ -32,18 +32,21 @@ blake2b_simd = "1.0.0"
 clap = { version = "4", features = ["derive", "color"] }
 log = "0.4"
 env_logger = "0.9"
-keccak = { version = "0.1", optional = true }
+# keccak = { version = "0.1", optional = true }
+keccak = { version = "0.1" }
 parking = { package = "parking_lot", version = "0.12", optional = true }
 shuttle = { version = "0.3", optional = true }
+chrono = "0.4.26"
 
 [features]
 default = ["parking_lot"]
 parking_lot = ["parking", "tokio/parking_lot"]
 bench = []
-no-asm = ["keccak"]
+no-asm = []
+# no-asm = ["keccak"]
 
-[target.'cfg(target_os = "windows")'.dependencies]
-keccak = "0.1"
+# [target.'cfg(target_os = "windows")'.dependencies]
+# keccak = "0.1"
```

Comment:
The right thing here is better feature gating over x86_64 (and consider adding an aarch64 implementation)

Reply:
Do you want me to create special conditions for `aarch64`?  The complexity of cfg() gates was a bit unclear, so I opted to go the path of "enable on all, unless".  The crate, if not used, should be optimized away in `--release` mode.  Otherwise `Cargo.toml` has to contain enable for windows and enable for macos where target arch == aarch64...  I felt that such complexity is better applied directly at the ASM import gates in the rust code, where it just loads wasm if needed, leaving toml alone...

Reply:
This should be now fixed in `main`